src/components/MemberContribution/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface TaskDetails {
+  title: string;
+  state: string;
+  createdAt: string;
+  updatedAt: string;
+  url: string;
+  raisedBy: string;
+  featureUrl: string;
+  purpose:string
+  id:string
+  isNoteworthy:string
+}
+
+export type ContributionAccordionItemProps = Readonly<{
+  task: TaskDetails;
+  isTitle: boolean;
+  openTaskStatusUpdateModal: (taskId: string, isTaskNoteworthy: string) => void;
+}>;
```

Comment:
- if the props are specific to `ContributionAccordianItem` component let's keep it together with the component

Reply:
I thought the code would look cleaner this way.  Keeping it in the same file for noe